### ISSUE TITLE
Link blue does not have enough contrast in most contexts

### ISSUE THREAD

**Issue by NomeQ** [Trillium-Solutions/mendocino-wp-theme#23]
The basic link color should be darker.

**Comment by NomeQ** [comment]
Needs to be darker everywhere - too low of contrast on front page alerts/news links:
<img width="359" alt="Screen Shot 2021-02-05 at 10 46 15 AM" src="https://user-images.githubusercontent.com/9342771/107075796-75dfe300-679f-11eb-9c12-6103d86031f3.png">
